package.json

```diff
@@ -30,7 +30,7 @@
   "dependencies": {
     "@pinia/testing": "^0.0.14",
     "@vcmap/core": "^5.0.0-rc.25",
-    "cooperation_jn_common_ui": "github:sigrennesmetropole/cooperation_jn_common_ui",
+    "cooperation_jn_common_ui": "github:sigrennesmetropole/cooperation_jn_common_ui#82611a7",
```

Comment:
Use the main branch instead of specific commit

Reply:
discussed on slack, all is good :)